mmv1/products/modelarmorglobal/Floorsetting.yaml

```diff
@@ -0,0 +1,253 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
```

Comment:
The generated [import_format](https://github.com/modular-magician/terraform-provider-google-beta/compare/d120ef7f8e3c8ccce304885105c7ca92ee22f4f0..429583f093808d7a0a5ebbaedb2bf014c6a4ccd1#diff-c0b945a6658cc1f93055fc86314539b76c25a468f18890caba8243567aa1abd5R328) is as following:
```
* `{{parent}}/locations/{{location}}/floorSetting/{{name}}`
* `{{parent}}/{{location}}/{{name}}`
```

I am not sure what value is expected for `name`.

I think you can customize `import_format` in this yaml file.

```
import_format:
  - '{{parent}}/locations/{{location}}/floorSetting'
```


Please let me know if you have questions.